content/_layouts/redirect.html.haml

```diff
@@ -0,0 +1,37 @@
+- # Define a redirect elsewhere with optional support for browser language detection
+- # Configuration is via front matter, one URL is the non-JS default, others are redirected to via JS if language matches
+- #
+- # ---
+- # layout: redirect
+- # redirect_url: "https://wiki.jenkins-ci.org/display/JENKINS/Remote+access+API"
+- # localized_urls:
+- #   ja: "https://wiki.jenkins-ci.org/display/JA/Remote+access+API"
+- #   de: "https://wiki.jenkins-ci.org/display/DE/Remote+access+API"
+- # ---
+
+
+:ruby
+  single_url = if page.localized_urls then false else true end
+
+
+%html
+  %head
+    %meta{:'http-equiv' => 'content-type', :content => 'text/html; charset=utf-8'}/
+    - if single_url
+      %meta{:'http-equiv' => 'refresh', :content => "0;URL=#{page.redirect_url}"}/
+    - else
+      %noscript
+        %meta{:'http-equiv' => 'refresh', :content => "0;URL=#{page.redirect_url}"}/
+  %body
+    %center
+      This content has moved to
+      %a{:href => page.redirect_url}
+        = page.redirect_url
+    %script
+      var language = window.navigator.userLanguage || window.navigator.language;
```

Comment:
Right, Accept-Language is an HTTP header which we have no access to.

However, when I configured Firefox for a few different languages (`intl.accept_languages`) that also affect Accept-Language, it switched the redirect over as expected. So my expectation is that this is Close Enough™.